<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>nifsClass</class>
 <widget class="QWidget" name="nifsClass">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1454</width>
    <height>855</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>nifs</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0,0,0);
border-color: rgb(0, 0, 0);
border-left-color: rgb(0, 0, 0);
border-right-color: rgb(0, 0, 0);
border-bottom-color: rgb(0, 0, 0);
border-top-color: rgb(0, 0, 0);
selection-color: rgb(84, 84, 84);
selection-background-color: rgb(71, 71, 71);</string>
  </property>
  <widget class="opengl" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>50</y>
     <width>981</width>
     <height>771</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
  </widget>
  <widget class="QToolBox" name="toolBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>441</width>
     <height>781</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(124, 124, 124);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="currentIndex">
    <number>10</number>
   </property>
   <widget class="QWidget" name="page">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>439</width>
      <height>482</height>
     </rect>
    </property>
    <attribute name="label">
     <string>Mesh</string>
    </attribute>
    <widget class="QTreeWidget" name="treeWidgetMesh">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>411</width>
       <height>401</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>439</width>
      <height>482</height>
     </rect>
    </property>
    <attribute name="label">
     <string>Materials</string>
    </attribute>
    <widget class="QTreeWidget" name="treeWidgetMaterial">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>411</width>
       <height>401</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>439</width>
      <height>482</height>
     </rect>
    </property>
    <attribute name="label">
     <string>Skeleton</string>
    </attribute>
    <widget class="QTreeWidget" name="treeWidgetSkeleton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>411</width>
       <height>401</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>439</width>
      <height>482</height>
     </rect>
    </property>
    <attribute name="label">
     <string>Animation</string>
    </attribute>
    <widget class="QTreeWidget" name="treeWidgetAnimation">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>411</width>
       <height>401</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="page_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>439</width>
      <height>482</height>
     </rect>
    </property>
    <attribute name="label">
     <string>Particles</string>
    </attribute>
    <widget class="QTreeWidget" name="treeWidgetParticles">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>411</width>
       <height>401</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="page_6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>439</width>
      <height>482</height>
     </rect>
    </property>
    <attribute name="label">
     <string>Havok</string>
    </attribute>
    <widget class="QTreeWidget" name="treeWidgetHavok">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>411</width>
       <height>401</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="page_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>439</width>
      <height>482</height>
     </rect>
    </property>
    <attribute name="label">
     <string>Misc</string>
    </attribute>
    <widget class="QTreeWidget" name="treeWidgetMisc">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>411</width>
       <height>401</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="page_8">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>439</width>
      <height>482</height>
     </rect>
    </property>
    <attribute name="label">
     <string>Textures</string>
    </attribute>
    <widget class="QTreeWidget" name="treeWidgetTextures">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>411</width>
       <height>401</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>370</y>
       <width>181</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_9">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>439</width>
      <height>482</height>
     </rect>
    </property>
    <attribute name="label">
     <string>File</string>
    </attribute>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>421</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Load file</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>421</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Save file</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>421</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Load image</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_10">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>439</width>
      <height>482</height>
     </rect>
    </property>
    <attribute name="label">
     <string>Options</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="page_11">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>439</width>
      <height>482</height>
     </rect>
    </property>
    <attribute name="label">
     <string>Render</string>
    </attribute>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>421</width>
       <height>241</height>
      </rect>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1651</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(124, 124, 124);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>830</y>
     <width>1651</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(124, 124, 124);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>opengl</class>
   <extends>QWidget</extends>
   <header>opengl.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="nifs.qrc"/>
 </resources>
 <connections/>
</ui>
